File: LAPORAN PRATIKUM/MODUL6/CODEBLOKS DAN PYTHON/SOAL 2/PRAK502-2210817210014-MuhammadQoharyRawidafany.py
def hitung (nilai1,nilai2):
    return (nilai1 - nilai2)
def mutlak (nilai):
    return (abs(nilai))
a,b,c,d = map(int,input().split())
hasil = hitung(a,c) + hitung(b,d)
print(mutlak(hasil))

File: LAPORAN PRATIKUM/MODUL6/CODEBLOKS DAN PYTHON/SOAL 2/PRAK502-2210817210014-MuhammadQoharyRawidafany.c
#include <stdio.h>
#include <math.h>
int hitung(int nilai1,int nilai2){
    return (nilai1-nilai2);
}
int mutlak(int nilai){
    return abs(nilai);
}
int main()
{
    int a,b,c,d,hasil;
    scanf("%d",&a);
    scanf("%d",&c);
    scanf("%d",&b);
    scanf("%d",&d);
    hasil = hitung(a,b) + hitung(c,d);
    printf("%d",mutlak(hasil));
    return 0;
}
